wampy/messages/result.py

```diff
@@ -43,4 +43,6 @@ def message(self):
 
     @property
     def value(self):
+        if self.yield_kwargs == None:
```

Comment:
should use `is` when comparing to a Singleton please. and then you can either just `return` or explicitly `return None`.

Reply:
Thruway returns values in the yield_args dictionary. yield_kwargs appears to be empty.

Reply:
hiya. i was just referring to the comparison to `None`, which is a Singleton. It's more pythonic to do `if self.yield_kwargs is None`.